src/assets/css/filesystem.css

```diff
@@ -1,7 +1,7 @@
 section#filesystem {
     position: relative;
     top: -0.925vh;
-    width: 43vw;
+    width: 30vw;
```

Comment:
Correct me if I'm wrong, but adapting the UI for larger screen ratios by changing the values like that should break it on more common screens, which is not something we want!

You should apply your changes by overriding the CSS in a media query rule, in [this file](https://github.com/GitSquared/edex-ui/blob/master/src/assets/css/extra_ratios.css).

Just copy your changes there in the correct media rule, and it should be fine.

Reply:
I tested this code in multiple aspects ratios and it works as expected. But I will follow your advice, move my changes to the recommended location and make sure it works.